airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -384,10 +384,6 @@ definitions:
         examples:
           - "{{ response.data.has_more is false }}"
           - "{{ 'next' not in headers['link'] }}"
-      decoder:
```

Comment:
Is removing a field a breaking change? Assuming a manifest.yaml declares this (even though they don't need to), would the parsing of the manifest fail?